core/server/master/src/test/java/alluxio/master/journal/ufs/UfsJournalLogWriterTest.java

```diff
@@ -360,14 +360,29 @@ private long writeJournalEntries(UfsJournalLogWriter writer, long startSN,
    *
    * @param writer the {@link UfsJournalLogWriter} instance for which the mock is created
    * @return the created mock {@link DataOutputStream} instance
+   * @throws IOException
    */
-  private DataOutputStream createMockDataOutputStream(UfsJournalLogWriter writer) {
+  private DataOutputStream createMockDataOutputStream(UfsJournalLogWriter writer)
+      throws IOException {
+    flushOutputStream(writer);
```

Comment:
Why do we need this call?

Reply:
Need, `flushOutputStream` ensures that the data is flushed to disk.